test/validators.js

```diff
@@ -3604,16 +3604,39 @@ describe('Validators', () => {
       });
     });
 
+    ['sha256', 'ripemd256'].forEach((algorithm) => {
+      test({
+        validator: 'isHash',
+        args: [algorithm],
+        valid: [
+          '2cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b9824',
+          '1d996e033d612d9af2b44b70061ee0e868bfd14c2dd90b129e1edeb7953e7985',
+          '80f70bfeaed5886e33536bcfa8c05c60afef5a0e48f699a7912d5e399cdcc441',
+          '579282cfb65ca1f109b78536effaf621b853c9f7079664a3fbe2b519f435898c',
+          '2CF24dba5FB0a30e26E83b2AC5b9E29E1b161e5C1fa7425E73043362938b9824',
+          '80F70bFEAed5886e33536bcfa8c05c60aFEF5a0e48f699a7912d5e399cdCC441',
+        ],
+        invalid: [
+          'KYT0bf1c35032a71a14c2f719e5a14c1',
+          'KYT0bf1c35032a71a14c2f719e5a14c1dsjkjkjkjkkjk',
+          'q94375dj93458w34',
+          '39485729348',
+          '%&FHKJFvk',
+        ],
+      });
+    });
+
+
     test({
       validator: 'isHash',
-      args: ['sha256'],
```

Comment:
Why remove this?

Reply:
I have grouped that with another alogirithm as it and another had same characteristics